crates/humanode-rpc/src/lib.rs

```diff
@@ -136,14 +149,14 @@ pub struct Deps<C, P, BE, VKE, VSF, A: ChainApi, SC> {
 }
 
 /// Instantiate all RPC extensions.
-pub fn create<C, P, BE, VKE, VSF, A, SC>(
+pub fn create<C, P, BE, VKE, VSF, A, SC, EC>(
     deps: Deps<C, P, BE, VKE, VSF, A, SC>,
 ) -> Result<RpcModule<()>, Box<dyn std::error::Error + Send + Sync>>
 where
     BE: Backend<Block> + 'static,
     BE::State: StateBackend<sp_runtime::traits::HashFor<Block>>,
-    C: ProvideRuntimeApi<Block> + BlockBackend<Block> + StorageProvider<Block, BE> + AuxStore,
-    C: BlockchainEvents<Block>,
+    C: ProvideRuntimeApi<Block> + CallApiAt<Block> + BlockBackend<Block>,
+    C: StorageProvider<Block, BE> + AuxStore + BlockchainEvents<Block>,
     C: HeaderBackend<Block> + HeaderMetadata<Block, Error = BlockChainError> + 'static,
```

Comment:
```suggestion
    C: ProvideRuntimeApi<Block> + CallApiAt<Block>,
    C: BlockBackend<Block> + StorageProvider<Block, BE> + AuxStore
    C: BlockchainEvents<Block>,
    C: HeaderBackend<Block> + HeaderMetadata<Block, Error = BlockChainError> + 'static,
```

Reply:
Have drafted this roughly in the UI, the idea is we group bounds by the source of where the trait comes from